desc/compute/_stability.py

```diff
@@ -200,3 +200,165 @@ def _magnetic_well(params, transforms, profiles, data, **kwargs):
         / (data["V_r(r)"] * data["<B^2>"])
     )
     return data
+
+
+@register_compute_fun(
+        name="gds2",
+        label="\\mathrm{gds2}",
+        units="~",
+        units_long="None",
+        description="Fieldline bending term1",
+        dim=3,
+        params=[],
+        transforms={"grid": []},
+        profiles=[],
+        coordinates="rtz",
+        data=[
+            "grad(alpha)",
+            ]
+        )
+def _gds2(params, transforms, profiles, data, **kwargs):
+    data["gds2"]= dot(data["grad(alpha)"], data["grad(alpha)"])
+
+
+@register_compute_fun(
+        name="gds21",
+        label="\\mathrm{gds21}",
+        units="~",
+        units_long="None",
+        description="Fieldline bending term2",
+        dim=3
+        params=[]
+        transforms={"grid": []},
+        profiles=[],
+        coordinates="rtz",
+        data=[
+            "grad(alpha)",
+            "grad(rho)"
+            ]
+        )
+def _gds21(params, transforms, profiles, data, **kwargs):
+    data["gds21"]= dot(data["grad(alpha)"], data["grad(rho)"])
+
+
+@register_compute_fun(
+        name="gds22",
+        label="\\mathrm{gds22}",
+        units="~",
+        units_long="None",
+        description="Fieldline bending term3",
+        dim=3
+        params=[]
+        transforms={"grid": []},
+        profiles=[],
+        coordinates="rtz",
+        data=[
+            "grad(rho)"
+            ]
+        )
+def _gds21(params, transforms, profiles, data, **kwargs):
+    data["gds22"]= dot(data["grad(rho)"], data["grad(rho)"])
+
+
+@register_compute_fun(
+        name="cvdrift",
+        label="\\mathrm{curvature-drift}",
+        units="~",
+        units_long="None",
+        description="Geometric part of the curvature drift"+"used for local stability analyses",
+        dim=3
+        params=[]
+        transforms={"grid": []},
+        profiles=[],
+        coordinates="rtz",
+        data=[
+            "dp_dpsi",
+            "psi_r",
+            "B",
+            "gbdrift",
+            "dpsidr"
+            ]
+        )
+def _cvdrift(params, transforms, profiles, data, **kwargs):
+    data["cvdrift"]= 1/data["B"]**2*data["dp_dpsi"] + data["gbdrift"]  
+
+
+@register_compute_fun(
+        name="gbdrift",
+        label="\\mathrm{gradB-drift}",
+        units="~",
+        units_long="None",
+        description="Geometric part of the gradB drift"+"used for local stability analyses",
+        dim=3
+        params=[]
+        transforms={"grid": []},
+        profiles=[],
+        coordinates="rtz",
+        data=[
+            "B",
+            "B_r",
+            "B_t",
+            "b",
+            "e^theta",
+            "e^zeta",
+            "iota",
+            "sqrt(g)",
+            "grad(alpha)"
+            "grad(B)"
```

Comment:
this is the derivative of the vector B (ie a 3x3 tensor), did you mean `grad(|B|)`?

Reply:
Yes. I'll change that.